lib/PipelineC/PipelineC.cpp

```diff
@@ -93,6 +95,8 @@ static void llvmErrorToRpError(llvm::Error Error, rp_error *Out) {
     *Out = rp_simple_error(Reason, "");
   };
 
+  // TODO: do we really need `handleAllErrors` here? It makes it seem as if
+  // we're overriding `*Out` a bunch of times, even though it only happens once.
```

Comment:
The `All` here means that if an error is unhandled it aborts. We're guaranteed that that doesn't happen because the second handler accepts the base class `llvm::ErrorInfoBase`.